dev-python/msgpack/msgpack-0.5.4.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy )
+
+inherit distutils-r1
+
+MY_PN="${PN}-python"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="MessagePack (de)serializer for Python"
+HOMEPAGE="http://msgpack.org https://github.com/msgpack/msgpack-python/ https://pypi.python.org/pypi/msgpack-python/"
+SRC_URI="mirror://pypi/${MY_PN:0:1}/${MY_PN}/${MY_P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="amd64 arm ~arm64 ppc x86"
+IUSE="+native-extensions test"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	native-extensions? (
+		>=dev-python/cython-0.16
+	)
+	test? (
+		dev-python/six[${PYTHON_USEDEP}]
+		dev-python/pytest[${PYTHON_USEDEP}]
+	)
+"
+
+S=${WORKDIR}/${MY_P}
+
+python_prepare_all() {
+	sed -i -e "/TRANSITIONAL = True/d" ./setup.py || die
```

Comment:
What does this do? Please leave a comment above it.

Reply:
transitional meant that by default the package name was "msgpack-python" instead of "msgpack". This changes in 0.5.6, which I'm going to add now instead, so that line in python_prepare_all is no longer necessary.